README.md

```diff
@@ -5,6 +5,11 @@ Ultra customizable hud featuring a unique and robust settings menu. Change eveyt
 
 ![Project Sloth Buttons 2](https://user-images.githubusercontent.com/91661118/170895968-c4224105-5c87-4947-af94-8e8bd792f566.png)
 
+### Unsupported ESX Version made by [reyyghi](https://github.com/reyyghi)
```

Comment:
Please only have a PR with these changes, the changes made to the README, everything else we will not merge into main branch.

Reply:
Okay, I understand